homeassistant/components/awair/manifest.json

```diff
@@ -6,5 +6,12 @@
   "codeowners": ["@ahayworth", "@danielsjf"],
   "config_flow": true,
   "iot_class": "cloud_polling",
-  "loggers": ["python_awair"]
+  "loggers": ["python_awair"],
+  "after_dependencies": ["zeroconf"],
```

Comment:
I'm not certain this is needed. Some components with zeroconf discovery declare it, others do not.

Reply:
you don't need it unless you import from it. Hassfest will tell you.